internal/metastore/db/dbcore/core.go

```diff
@@ -0,0 +1,105 @@
+package dbcore
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/milvus-io/milvus/internal/log"
+	"github.com/milvus-io/milvus/internal/util/paramtable"
+	"go.uber.org/zap"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+)
+
+var (
+	globalDB *gorm.DB
+)
+
+func Connect(cfg *paramtable.MetaDBConfig) error {
+	// load config
+	dsn := fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?charset=utf8mb4&parseTime=True&loc=Local", cfg.Username, cfg.Password, cfg.Address, cfg.Port, cfg.DBName)
+
+	var ormLogger logger.Interface
+	if cfg.Base.Log.Level == "debug" {
```

Comment:
Do we need an extra config for DB log level?

Reply:
No need here, just reuse the log level setting of Milvus. Here if Milvus log level is debug, we print the sql statement, by default it will not only when execution error happens.